spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/RegisteredApp.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.controller.support;
+
+import org.springframework.cloud.dataflow.core.ApplicationType;
+
+/**
+ * @author Vinicius Carvalho
+ *
+ */
+public class RegisteredApp {
```

Comment:
We have `AppRegistrationResource` which sits in the `spring-cloud-dataflow-rest-resource` package.  

Need to consider how this new bulk unregister operation will be included in `AppRegistryOperations`.   These various support classes should move out of the `server.controller.support` package and into the `rest-client`.